README.md

```diff
@@ -1,2 +1,33 @@
 # mulungwishi-webhook
 Webhook for the [Mulungwishi Fertilizer Project](https://www.youtube.com/watch?v=qkBsuUHCFWo)
+
+If you are using Python 3.4, then cd into the mulungwishi-webhook folder and then create a virtual environment with the following command:
+
+	$ python -m venv flask
+
+For Python 2
+
+ 	$ python -m virtualenv flask
+
+Note that in some operating systems you may need to use python3 instead of python. The above command creates a private version of your Python interpreter inside a folder named flask.
+
+If you are using any other version of Python older than 3.4, then you need to download and install virtualenv.py before you can create a virtual environment. If you are on Mac OS X, then you can install it with the following command:
+
+    $ sudo easy_install virtualenv
+
+On Linux you likely have a package for your distribution. For example, if you use Ubuntu:
+
+	$ sudo apt-get install python-virtualenv
+
+Windows users have the most difficulty in installing virtualenv, so if you want to avoid the trouble then install Python 3.4. If you want to install virtualenv on Windows then the easiest way is by installing pip first, as explaned in this page. Once pip is installed, the following command installsvirtualenv`:
+
+	$ pip install virtualenv
```

Comment:
you don't need to duplicate information on how to install a virtualenv. just put a link to some documentation, or the virtualenv project.

what *is* needed then, is the list of steps: install the requirements,  and how to run the app